docs/jwst/tweakreg/README.rst

```diff
@@ -27,7 +27,28 @@ Alignment
 ---------
 The source catalogs for each input image are compared to each other
 and linear (affine) coordinate transformations that align these
-catalogs are derived.
+catalogs are derived.  This fit insures that all the input images
+are aligned in a 'relative' manner.  This results in a combined
+source catalog for the entire set of input images as if they were
+combined into a single mosaic.
+
+An astrometric reference catalog then gets generated by querying
+a GAIA-based astrometric catalog web service for all astrometrically
+measured sources in the combined field-of-view of the set of input
+images. This catalog gets generated from the catalogs available
+through the GSSS STScI web service at:
+
+.. _GSSSCATALOGS: http://gsss.stsci.edu/webservices/vo/CatalogSearch.aspx
+
+The combined source catalog derived from the input images
+then gets cross-matched and fit to this astrometric reference catalog
+to align the input images to the astrometric reference catalog's
+coordinate system.  The pipeline initially supports fitting to the
```

Comment:
I find this slightly not very clear. I would suggest eliminating "to align ..." - this is well explained in the last sentences.
```suggestion
then gets cross-matched and fit to this astrometric reference catalog.
The pipeline initially supports fitting to the
```

Reply:
Addressed in e6ae123.